Rocket.Chat/Extensions/EnumExtensions.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  EnumExtensions.swift
+//  Rocket.Chat
+//
+//  Created by Artur Rymarz on 18.04.2018.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+
+#if swift(>=4.2)
+fatalError("Remove CaseIterable. This should be implemented in Swift 4.2.")
+#else
```

Comment:
🤓

Reply:
🙈🙊 Could remove that or at least replace with just a warning 🤯